Tests/AssistantV1Tests/AssistantTests.swift

```diff
@@ -166,66 +160,9 @@ class AssistantTests: XCTestCase {
             // verify output
             XCTAssertTrue(message.output.logMessages.isEmpty)
             XCTAssertEqual(message.output.text, result1)
-            XCTAssertNotNil(message.output.nodesVisited)
-            XCTAssertEqual(message.output.nodesVisited!.count, 1)
-            XCTAssertNotNil(message.output.nodesVisitedDetails)
-            XCTAssertNotNil(message.output.nodesVisitedDetails!.first)
-            XCTAssertNotNil(message.output.nodesVisitedDetails!.first!.dialogNode)
-
-            context = message.context
             expectation1.fulfill()
         }
         waitForExpectations()
-
-        let description2 = "Continue a conversation."
-        let expectation2 = self.expectation(description: description2)
-
-        let input = MessageInput(text: "Turn on the radio.")
-        let result2 = ["Sure thing! Which genre would you prefer? Jazz is my personal favorite."]
-
-        assistant.message(workspaceID: workspaceID, input: input, context: context!) {
-            response, error in
-
-            if let error = error {
-                XCTFail(unexpectedErrorMessage(error))
-                return
-            }
-            guard let message = response?.result else {
-                XCTFail(missingResultMessage)
-                return
-            }
-
-            // verify input
-            XCTAssertEqual(message.input.text, input.text)
-
-            // verify context
-            XCTAssertEqual(message.context.conversationID, context!.conversationID)
-            XCTAssertNotNil(message.context.system)
-            XCTAssertNotNil(message.context.system!.additionalProperties)
-            XCTAssertFalse(message.context.system!.additionalProperties.isEmpty)
-
-            // verify entities
-            XCTAssertEqual(message.entities.count, 1)
-            XCTAssertEqual(message.entities[0].entity, "appliance")
-            XCTAssertEqual(message.entities[0].location[0], 12)
-            XCTAssertEqual(message.entities[0].location[1], 17)
-            XCTAssertEqual(message.entities[0].value, "music")
-
-            // verify intents
-            XCTAssertEqual(message.intents.count, 1)
-            XCTAssertEqual(message.intents[0].intent, "turn_on")
-            XCTAssert(message.intents[0].confidence >= 0.80)
-            XCTAssert(message.intents[0].confidence <= 1.00)
-
-            // verify output
-            XCTAssertTrue(message.output.logMessages.isEmpty)
-            XCTAssertEqual(message.output.text, result2)
-            XCTAssertNotNil(message.output.nodesVisited)
-            XCTAssertEqual(message.output.nodesVisited!.count, 3)
-
-            expectation2.fulfill()
-        }
-        waitForExpectations()
```

Comment:
Why are we deleting tests?

Reply:
This deleted a portion of a test that was very tightly coupled with a specific node in a test workspace that has been deleted. This makes the test more generic without hampering what we are really trying to test here.